src/System.CommandLine.Tests/SymbolResultTests.cs

```diff
@@ -9,6 +9,36 @@ namespace System.CommandLine.Tests
 {
     public class SymbolResultTests
     {
+
```

Comment:
I'm not sure this is the best class for this test. The defect was in SymbolResult but exposed via GetSuggestions.